src/skins/vector/css/matrix-react-sdk/views/messages/MTextBody.css

```diff
@@ -13,12 +13,3 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
-.mx_MTextBody {
-    white-space: pre-wrap;
```

Comment:
The use case is things like the `:     (` smiley or `            <-- idiot` without having to use backticks and make it formatted as monospace.  This is the expected behaviour for those from chat systems like IRC.  And if people do paste text from elsewhere, i'd expect them to want linefeeds preserved rather than HTMLified all into one great big paragraph.